base/gmp.jl

```diff
@@ -21,8 +21,16 @@ else
 end
 const CdoubleMax = Union{Float16, Float32, Float64}
 
-version() = VersionNumber(unsafe_string(unsafe_load(cglobal((:__gmp_version, :libgmp), Ptr{Cchar}))))
-bits_per_limb() = Int(unsafe_load(cglobal((:__gmp_bits_per_limb, :libgmp), Cint)))
+if Sys.iswindows()
+    const libgmp = "libgmp-10.dll"
```

Comment:
Can we pull in the version from the buildsystem? See base/Makefile the build_h.jl generation

Reply:
@gbaraldi bump

Reply:
I'm not sure, maybe we can pull the version from the jll from inside the makefile? It's kind of annoying not being able to depend on the jll proper here.

Reply:
In fact, is there a reason for BigNums being part of base instead of being stdlibs?

Reply:
They're used in Base, no?

Reply:
They’re needed quite early because we can parse literal bignums

Reply:
We can get version numbers from the makefile system, then embed those into build_h.jl. I suggest we try some awk trickery to figure out the so major version number.